redaxo/src/core/lib/list.php

```diff
@@ -345,6 +355,60 @@ public function getLinkAttributes($column, $default = null)
         return $this->linkAttributes[$column] ?? $default;
     }
 
+    // row attribute setter/getter etc
+
+    /**
+     * Methode, um dem der Zeile (<tr>) Attribute hinzuzufügen.
```

Comment:
```suggestion
     * Methode, um der Zeile (<tr>) Attribute hinzuzufügen.
```

Eigentlich schreiben wir die phpdocs übrigens in englisch (es gibt aber auch noch ein paar wenige Altreste in deutsch).